sc/src/main/java/uk/ac/ebi/atlas/experimentimport/ScxaExperimentDao.java

```diff
@@ -114,4 +117,13 @@ private String getPublicationIdsAsString(Collection<String> publicationIds) {
                         Collectors.joining(", "),
                         (result) -> result.isEmpty() ? null : result));
     }
+
+    public HashSet<String> getExperimentByGroupAccession(String experimentAccessionLike){
```

Comment:
You can rename this to something like `getExperimentByAccessionPattern(String experimentAccessionPattern)` to make its purpose a bit clearer.